common/lib/xmodule/xmodule/templates/html/zooming_image.yaml

```diff
@@ -1,10 +1,42 @@
 ---
 metadata:
-    display_name: Zooming Image
+    display_name: Zooming Image Tool
 data: |
-      <h2>ZOOMING DIAGRAMS</h2>
-      <p>Some edX classes use extremely large, extremely detailed graphics. To make it easier to understand we can offer two versions of those graphics, with the zoomed section showing when you click on the main view.</p>
-      <p>The example below is from <a href="https://www.edx.org/course/mit/7-00x/introduction-biology-secret-life/1014" target="_blank">7.00x: Introduction to Biology</a> and shows a subset of the biochemical reactions that cells carry out. </p>
+      <h2>Zooming Image Tool</h2>
+      <p>Use the Zooming Image Tool to enable learners to see details of large, complex images.</p>
+      <p>With the Zooming Image Tool, the learner can move the mouse pointer over a part of the image to enlarge it and see more detail.</p>
+      <p>To use the Zooming Image Tool, you must first add the <a href="http://files.edx.org/jquery.loupeAndLightbox.js" target="_blank">jquery.loupeAndLightbox.js JavaScript file</a> to your course.
+      <p>You must also add both the regular and magnified image files to your course.</p>
+      <p>The HTML format required to use the Zooming Image Tool is:
+        <pre>
+        &lt;div class="zooming-image-place" style="position: relative;"&gt;
+          &lt;a class="loupe" href="<i>path to the magnified version of the image</i>"&gt;
+            &lt;img alt="<i>alt text for screen readers</i>" 
+              src="<i>path to the image you want to display in the unit</i>" /&gt;
+          &lt;/a>
+          &lt;div class="script_placeholder" 
+            data-src="<i>path to the jquery.loupeAndLightbox.js JavaScript file in your course</i>" /&gt;
```

Comment:
(this is really hard to review without rendering... definitely need to look this over on stage)